message-switch.opam

```diff
@@ -9,7 +9,7 @@ tags: [ "org:xapi-project" ]
 build: [
   ["./configure" "--bindir" "%{bin}%"]
   ["dune" "build" "-p" name "-j" jobs]
-  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "runtest" "-j" jobs] {with-test}
```

Comment:
did you make sure that this didn't change the number of message switch tests that run?

Reply:
The only tests run before and after are using the shell script.